### ISSUE TITLE
Why rotate trajectory 10 times for denoise training?

### ISSUE THREAD

**Issue by JunrQ** [YvanYin/GoalFlow#21]
Dear author, thank you for your work. I have a question about the method `normalize_xy_rotation`, why rotate `gt_trajs_` 10 times, for denoise training?

**Comment by ZebinX** [comment]
@JunrQ We did not experiment with other rotation counts, so this setting was not specifically optimized.

**Comment by smilebetterworld** [comment]
@ZebinX , as mentioned in other issue, gt_trajs is rotated 10times, and type_embedding input dim is set 30（3*10）， so if type_embedding  input dim is set 3, is there any problem? because only 0~2 is used
self.type_embedding = nn.Embedding(30, self._config.tf_d_model)  --> self.type_embedding = nn.Embedding(3, self._config.tf_d_model)
